packages/node/src/evm/contracts/airnodeRrp.ts

```diff
@@ -8,8 +8,8 @@ const MadeFullRequest = ethers.utils.id(
   'MadeFullRequest(address,bytes32,uint256,uint256,address,bytes32,address,address,address,bytes4,bytes)'
 );
 
-const FulfilledRequest = ethers.utils.id('FulfilledRequest(address,bytes32,uint256,bytes)');
-const FailedRequest = ethers.utils.id('FailedRequest(address,bytes32)');
+const FulfilledRequest = ethers.utils.id('FulfilledRequest(address,bytes32,bytes)');
```

Comment:
Not in scope here, but I wonder if these could be generated from importing the contract ABI. i.e. something like:

```ts
// Not sure what the exact ABI import looks like
// Assuming events have the structure of "Name(...types)"
const eventTopics = airnodeRrpAbi.events.map((e) => {
  const name = e.name.split('(')[0];
  const topic = ethers.utils.id(e);
  return { name, topic };
});

export function getTopics() {
  // Build the same topic object
  return eventTopics.reduce((acc, event) => {
    return { ...acc, [event.name]: event.topic };
  }, {});
}
```

Then there wouldn't be any need to maintain the topics here

Reply:
That's a really good idea but do you still see this as something needed after we deploy the contracts to mainnet? 
For newer versions of the protocol maybe 🤔

Reply:
Probably not and it might actually be an over-optimization anyway. I just see these getting updated all the time, but they should probably settle in later versions. Ignore me 😁